crate/operator/upgrade.py

```diff
@@ -173,10 +174,38 @@ async def handle(  # type: ignore
         logger: logging.Logger,
         **kwargs: Any,
     ):
+        self.schedule_notification(
```

Comment:
Is it worth moving this to _after_ `await upgrade_cluster`?

Reply:
sure, we can do that.